charm-slurmd/src/charm.py

```diff
@@ -7,10 +7,11 @@
 from time import sleep
 
 from omnietcd3 import Etcd3AuthClient
+from singularity_ops import SingularityOps
 from ops.charm import CharmBase, CharmEvents
 from ops.framework import EventBase, EventSource, StoredState
 from ops.main import main
-from ops.model import ActiveStatus, BlockedStatus, WaitingStatus
+from ops.model import ActiveStatus, BlockedStatus, WaitingStatus, ModelError
```

Comment:
The linter is complaining about the order here, as it is not alphabetical.

```suggestion
from ops.model import ActiveStatus, BlockedStatus, ModelError, WaitingStatus
```